scripts/matlab/AutoscoperConnection.m

```diff
@@ -30,10 +30,10 @@ function closeConnection(obj)
 %             fclose(obj.socket_descriptor); % 
             fwrite(obj.socket_descriptor,[13]);
 %             delete(obj.socket_descriptor);
-            while obj.socket_descriptor.BytesAvailable == 0
-                pause(1)
-            end
-            data = fread(obj.socket_descriptor, obj.socket_descriptor.BytesAvailable);
+%             while obj.socket_descriptor.BytesAvailable == 0
+%                 pause(1)
+%             end
+%             data = fread(obj.socket_descriptor, obj.socket_descriptor.BytesAvailable);
```

Comment:
This shouldn't be necessary to remove, since the [socket confirms](https://github.com/BrownBiomechanics/Autoscoper/blob/main/autoscoper/src/net/Socket.cpp#L307) that it received the close connection command (by responding with a confirmation of 13) before shutting down the connection.